mws/acceptance/vpc_endpoint_test.go

```diff
@@ -0,0 +1,26 @@
+package acceptance
+
+import (
+	"os"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/internal/acceptance"
+)
+
+func TestMwsAccVpcEndpoint(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	if cloudEnv != "MWS" {
+		t.Skip("Cannot run test on non-MWS environment")
+	}
+	acceptance.Test(t, []acceptance.Step{
+		{
+			Template: `resource "databricks_mws_vpc_endpoint" "my_vpc_endpoint_relay" {
```

Comment:
and insert 
```
data "aws_caller_identity" "current" {}

```

in the beginning of template.

Reply:
rename `my_vpc_endpoint_relay` -> `this`

Reply:
done